src/main/java/org/springframework/data/elasticsearch/client/elc/RequestConverter.java

```diff
@@ -1646,7 +1647,7 @@ private void prepareNativeSearch(NativeQuery query, MultisearchBody.Builder buil
 	}
 
 	@Nullable
-	private co.elastic.clients.elasticsearch._types.query_dsl.Query getQuery(@Nullable Query query,
+	public co.elastic.clients.elasticsearch._types.query_dsl.Query getQuery(@Nullable Query query,
```

Comment:
```suggestion
	co.elastic.clients.elasticsearch._types.query_dsl.Query getQuery(@Nullable Query query,
```
changing the visibility to package scope is enough here, don't expose it to public